### ISSUE TITLE
tcptop requires node and pod parameteres

### ISSUE THREAD

**Issue by mauriciovasquezbernal** [kinvolk/inspektor-gadget#260]
tcptop should be able to work using only the pod name. Inspektor Gadget could determine the node where the pod is running by using the api server. 

Current status:

```
./kubectl-gadget-linux-amd64 tcptop --pod mypod
FATA[0000] tcptop only works with --node and --podname   args="[mypod]" command="kubectl-gadget tcptop
```

Desired status:

```
./kubectl-gadget-linux-amd64 tcptop --pod mypod
... works fine 
```

**Comment by eiffel-fl** [comment]
I added the `wontfix` due to problem described [here](https://github.com/kinvolk/inspektor-gadget/pull/448#issuecomment-1011316064)

**Comment by eiffel-fl** [comment]
This is normally fixed by #555!
